server/routes/assess.ts

```diff
@@ -44,5 +49,13 @@ export default function applyRoutes(controllers: Controllers, router: Router, se
     auditEvent: 'CREATE_APPLICATION_NOTE_AS_ASSESSOR',
   })
 
+  get(paths.assessmentDetails.show.pattern, assessmentsController.show(), {
+    auditEvent: 'VIEW_ASSESSMENT_SHOW',
+  })
+
+  post(paths.assessmentDetails.update.pattern, assessmentsController.update(), {
+    auditEvent: 'VIEW_ASSESSMENT_UPDATE',
```

Comment:
Don't know if this is really important, but should this just be `ASSESSMENT_UPDATE` or is it useful in audit events to have the page the person is viewing when they take the action?

Reply:
You're right, `View` doesn't make sense in this scenario 👍🏾